src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -10,4 +10,5 @@
     <include file="db/changelog/db.changelog-entities.xml"/>
     <include file="db/changelog/db.changelog-references.xml"/>
     <include file="db/changelog/db.changelog-indexes.xml"/>
+    <include file="db/changelog/db.changelog-defaultData.xml"/>
```

Comment:
This kind of data usually called seed https://en.wikipedia.org/wiki/Database_seeding
Select appropriate name i.e. db.changelog.stepSuggestionSeed.xml

Reply:
fixed